doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -10,88 +10,87 @@ sort_idx: 60
 * TOC
 {:toc}
 
-The Kaa Configuration subsystem is responsible for configuring endpoints by supplying them with the structured data of user-defined complexity that is managed
-via the Kaa server. The fact that Kaa operates with the uniformly structured data deserves a special emphasis. By knowing the data structure of the
-application, Kaa provides a number of very useful features, such as:
+To configure an [endpoint]({{root_url}}Glossary/#endpoint-ep), the Kaa **Configuration subsystem** sends structured data from [Kaa server]({{root_url}}Glossary/#kaa-server) to that endpoint.
+The data structure is defined in the schema selected by the user.
+For more information about schemas in Kaa, see [Common type library]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library).
 
-* Automatic generation of the configuration object model in the endpoint SDK
-* Automatic generation of the default configuration
-* Enforcement of the configuration integrity
-* Endpoint-specific configuration view that is based on the endpoint membership in endpoint groups
+Since Kaa operates on uniformly structured data, you can use the following features:
 
-# Configuration management
+* Automatic generation of the configuration object model in the endpoint SDK.
+* Automatic generation of the default configuration.
+* Enforcement of the configuration integrity.
+* Endpoint-specific configuration view based on the endpoint group membership.
 
-The structure of the configuration data is determined by the customizable configuration schema. 
-It is the responsibility of the Kaa developer to construct the configuration schema and make the client application interpret the data supplied by the endpoint SDK.
-Kaa developer can provision the configuration schema using Administration UI (see image below) or [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Configuration/saveConfigurationSchema).
+## Updating configuration data
+
+To provision a configuration schema:
+
+- Use the [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Configuration/saveConfigurationSchema).
+
+OR
+
+- Use the [Administration UI]({{root_url}}Glossary/#administration-ui) to open the **Configuration** page under the **Schemas** section of your application and click **Add schema**.
 
```

Comment:
Procedure of adding configuration schema is outdated. This part should be updated according to new procedure based on CTL.